src/app/services/siteConstants.ts

```diff
@@ -7,3 +7,5 @@ export const SITE_SUBJECT_TITLE = "Computer Science";
 export const WEBMASTER_EMAIL = "webmaster@isaaccomputerscience.org";
 
 export const TEACHER_REQUEST_ROUTE = "/pages/teacher_accounts";
+
+export const GAMEBOARD_SUBJECT = ["compsci"]
```

Comment:
This no longer needs to be an array. Making it a string will simplify the places where this is used. If it's too much work for this ticket, then creating another ticket to address this simplification would be the right thing to do.